src/lib/PreviewUI.js

```diff
@@ -369,11 +369,14 @@ class PreviewUI {
      */
     setupHeader(headerTheme, logoUrl) {
         const headerContainerEl = this.container.querySelector(SELECTOR_BOX_PREVIEW_HEADER_CONTAINER);
-        headerContainerEl.classList.remove(CLASS_HIDDEN);
-
+        const customLogoEl = this.container.querySelector(SELECTOR_BOX_PREVIEW_LOGO_CUSTOM);
         const headerEl = headerContainerEl.firstElementChild;
+        const defaultLogoEl = headerEl.querySelector(SELECTOR_BOX_PREVIEW_LOGO_DEFAULT);
+
+        headerContainerEl.classList.remove(CLASS_HIDDEN);
         headerEl.className = `${CLASS_BOX_PREVIEW_HEADER} ${CLASS_BOX_PREVIEW_BASE_HEADER}`;
         this.previewContainer.classList.add(CLASS_BOX_PREVIEW_HAS_HEADER);
+        defaultLogoEl.setAttribute('aria-label', __('logo'));
```

Comment:
It looks like the attribute we're using is inconsistent. Should these both use `alt` or `aria-label`?

Reply:
for `defaultLogoEl` it has to be `aria-label` because its NOT an `img` element

Reply:
but is mandatory for ALL the `img` elements to have an `alt` attribute

Reply:
Good catch. Should we add `role="img"` to the SVG, while we're at it?

Reply:
not because ScreenReader would never reach the svg, they will focus the container element that's `defaultLogoEl`

Reply:
but we can add `role=img` to `<span class="bp-default-logo">`